NEWS.md file added (#66)
Appsilon/data.validator: NEWS.md

# data.validator 0.2.0

- `validate_cols()` and `validate_rows()` will use all columns in dataframe if no column is passed
- Added a defensive wrapper (`data.validator:::check_assertr_expression()`) to add evaluation error to regular validation errors
- Removed deprecated `.data` calls in `dplyr::select()`
- `shiny.semantic::accordion()` bug fixes

# data.validator 0.1.6

> - No changes in this release. It was released to revive `data.validator` on CRAN, previously archived due to the archiving of `shiny.semantic` dependency.

Reviewer: ```suggestion
No changes. Released to restore `data.validator` on CRAN, previously archived due to the archiving of `shiny.semantic` dependency.
```

> # data.validator 0.2.0

Reviewer: Misses bugfix: https://github.com/Appsilon/data.validator/pull/48
MIsses fix: https://github.com/Appsilon/data.validator/pull/54

> -   `validate_cols()` will use all columns in dataframe if no column is passed

Reviewer: `validate_rows` was also changed in the same PR: https://github.com/Appsilon/data.validator/pull/53

-   Added a defensive wrapper (`data.validator:::check_assertr_expression()`) to add evaluation error to regular validation errors
-   New examples added:
    -   Shiny App example demo example added
    -   Usage examples added in README
    -   Connect workflow examples added

> -   pkgdown documentation updated

Reviewer: Not worth mentioning.

# data.validator 0.1.6

-   README updated
-   R version updated
    -   windows: `3.6` to `4.1.2`
    -   macOS: `4.0.2` to `4.1.2`

>     -   ubuntu 18.04: `3.6` to `4.1.2`

Reviewer: This is not a change that affects the code. Version 0.1.6 was pushed to CRAN only to restore `data.validator` (after one of its dependencies was removed) and there were no actual changes implemented. The changelog should reflect that.

# data.validator 0.1.5

-   `assert_cols`, `assert_rows` and `assert_if` renamed to `validate_cols`, `validate_rows` and `validate_if`

> -   Documentation updated for newly added functions.

Reviewer: Not worth mentioning.

# data.validator 0.2.0

-   `validate_cols()` and `validate_rows()` will use all columns in dataframe if no column is passed

-   Added a defensive wrapper (`data.validator:::check_assertr_expression()`) to add evaluation error to regular validation errors

-   New examples added:

    -   Shiny App example demo example added
    -   Usage examples added in README

>     -   Connect workflow examples added

Reviewer: Not worth mentioning.

-   Removed deprecated `.data` calls in `dplyr::select()`

-   `shiny.semantic::accordion()` bug fixes

# data.validator 0.1.6

> -   Dependencies updated

Reviewer: Please describe the purpose of this release, not a vague "dependencies update".

Author: Only package dependency change I was able to find in #25 was related to ROxygen version upgrade.  So I've added this change in NewsMD file.

Reviewer: As I wrote in my previous comment: Version 0.1.6 was pushed to CRAN only to restore data.validator (after one of its dependencies was removed) and there were __no actual changes__ implemented. The changelog should reflect that - you need to write that there were __no changes__ and describe why (in a super condensed way). Simply read the comments in the PR you linked here to learn more (shiny.semantic was down and all packages that depended on it were also removed from CRAN).
